### PR TITLE
MGMT-14906: Enable ODF Support and Prepare for OCS Removal

### PR DESCRIPTION
**Pull request by danmanor**
Add ODF support to replace OCS and make necessary adjustments for a smooth transition.

### CODE REVIEW

File: src/assisted_test_infra/test_infra/utils/oc_utils.py [hunk 1]
```diff
@@ -19,13 +19,21 @@ def extend_parser_with_oc_arguments(parser):
...
+    # to be removed
     parser.add_argument(
         "-ocs",
         "--oc-server",
         help="Server for oc login, required if --oc-token is provided",
         type=str,
         default="https://api.ocp.prod.psi.redhat.com:6443",
     )
+    parser.add_argument(
+        "-odf",
```
Comment (osherdp) [inline src/assisted_test_infra/test_infra/utils/oc_utils.py:31]: ?
I don't think ``ocs-`` above refers to OCS, it's just an acronym for oc server

**Comment by osherdp** [comment]
It's worth syncing with QE to make sure they remove the traces from their code

File: src/assisted_test_infra/test_infra/utils/oc_utils.py (hunk 1 above)
Comment (danmanor) [reply src/assisted_test_infra/test_infra/utils/oc_utils.py]: For sure, thanks

**Review by osherdp — APPROVED** [review]

**Pull request merged**